src/patterns/components/slideshow/base.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="u-bgGray u-posRelative Slideshow js-Slideshow u-pullSides1">
+  <h2 class="u-hiddenVisually">Fun facts about Aileen</h2>
+  <ul class="u-flex u-listUnstyled Slideshow-slides js-Slideshow-slides u-padEnds2"  id="Slideshow-slides">
+    {{#each drizzle.data.fun-facts.contents.aileen}}
+      <li class="Grid Grid--withGutter Grid--alignMiddle Slideshow-slide js-Slideshow-slide u-flexExpand u-flexJustifyCenter u-sm-flexJustifyStart u-padSides5 {{#if @first}}is-visible{{/if}}">
```

Comment:
I think it could go either way. The one benefit it provides is the screenreader says how many items are in the section when you enter it. And, it _is_ a list of things, so I am inclined to keep it.